packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -638,16 +643,16 @@ export const RemoteServerFactory = {
         const encodedSlasCredentials = Buffer.from(`${clientId}:${clientSecret}`).toString('base64')
 
         app.use(
-            SLAS_CUSTOM_PROXY_PATH,
+            getSLASPrivateProxyPath(),
```

Comment:
Nit: we should have create a variable for this one here to avoid calling the func many places. 
```
const slasPrivateProxyPath = getSLASPrivateProxyPath()
```

Reply:
I don't think we call _createApp more than once per server so I am not sure if creating a temp variable there will change much.

However, this is good input as we probably want to memoize 2 things in the subsequent PR when we implement getNamespace:
* the `getNamespace` lookup itself, so we are not having to read the config file all the time
* the asset publicPath inside the getAssetUrl function of pwa-kit-react-sdk `universal/utils`. This seems to be called whenever we fetch an asset and we don't want to have to compute the publicPath (which resolves to `(/somenamespace)/mobify/bundle/(bundleId|development)` on the server) each time

Reply:
@vcua-mobify, I think it is still worth for this particular func because you are calling it 3 times in the same method. Creating a variable at the top of the method will help to reduce the amount of change in case we change the function name. 

```
_setupSlasPrivateClientProxy(app, options) {
   const slasPrivateProxyPath = getSLASPrivateProxyPath()
}
```

Reply:
Done! The calls to `getSLASPrivateProxyPath()` have been reduced via a variable.